homework2/fastq_filtrator.py

```diff
@@ -0,0 +1,56 @@
+def length_check(read, bounds):
+    if bounds == (0, 2 ** 32):
+        return True
+    length = len(read[1].rstrip('\n'))
+    if len(bounds) == 2:
+        return (length > bounds[0] and length < bounds[1])
+    else:
+        return (length < bounds)
+
+def gc_check(seq, gc_bounds):
+    if gc_bounds == (0, 100):
+        return True
+    seq = seq.rstrip('\n').upper()
+    GC_count = (seq.count('G') + seq.count('C')) * 100 / len(seq)
+    if len(gc_bounds) == 2:
+        return (GC_count > gc_bounds[0] and GC_count < gc_bounds[1])
+    else:
+        return (GC_count < gc_bounds)
+
+def quality_check(read, quality_threshold):
+    if quality_threshold == 0:
+        return True
+    total_phred33 = 0
+    for symbol in (read[3].rstrip('\n')):

```

Comment:
`(rl)strip` по-умолчанию убирают пробельные символы (в т.ч. переносы строк)
```suggestion
    for symbol in (read[3].rstrip()):

```